### PR TITLE
RDoc-1159 Studio > Database > Tasks > Ongoing Tasks > RavenDB ETL

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: Documentation/4.0/Raven.Documentation.Pages/server/ongoing-tasks/etl/basics.markdown [hunk 1]
```diff
@@ -1,116 +1,145 @@
...
-A typical example would be a product catalog that is shared among multiple tenants where each tenant can modify the products or add new ones.
+{PANEL: Why use ETL}
 
-Another example is the reporting database. RavenDB is a wonderful database for OLTP scenarios but for reporting you might use an existing solution working on
-a relational database. In order to push RavenDB data there we need to transform it to match the relational model.
+* **Share relevant data**  
+  Data that needs to be shared can be sent in a well-defined format matching your requirements so that only relevant data is sent.  
 
+* **Protect your data - Share partial data**  
+  Limit access to sensitive data, details that should remain private can be filtered out as you can sare partial data.
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/server/ongoing-tasks/etl/basics.markdown]: sare -> share

File: Documentation/4.0/Raven.Documentation.Pages/server/ongoing-tasks/etl/basics.markdown [hunk 2]
```diff
@@ -1,116 +1,145 @@
...
+  Limit access to sensitive data, details that should remain private can be filtered out as you can sare partial data.  
 
-## ETL Tasks
+* **Reduce system calls**  
+  Distribute data to related services in your system architecture so that they have their _own copy_ of the data and can access it without making a cross-service call.  
+  i.e. A product catalog can be shared among multiple stores where each can modify the products or add new ones.  
 
-As mentioned, RavenDB supports two types of ETL destinations:
+* **Transform the data**  
+  * Modify content sent as needed with a javascript script.
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/server/ongoing-tasks/etl/basics.markdown]: javascript script -> JavaScript code?

File: Documentation/4.0/Raven.Documentation.Pages/server/ongoing-tasks/etl/basics.markdown [hunk 3]
```diff
@@ -1,116 +1,145 @@
...
-The functions specific for ETL:
+In addition to ECMAScript 5.1 API, RavenDB introduces the following functions and members:  
 
-| ------ |:------:| ------ |
-| `loadTo<Target>(obj)` | function | Indicates to load an object to a specified `<Target>`. The target must be either the name of a collection (RavenDB ETL) or a table (SQL ETL). |
-| `loadAttachment(name)` | function | Loads an attachment (SQL ETL only) |
+| ------ | ------ | ------ |
+| `this` | object | The current document (with metadata) |
+| `id(document)` | function | Returns the document ID |
+| `load(id)` | function | Load another document.<br/>This will increase the maximum number of allowed steps in a script.<br/>**Note**: Changes made to the other _loaded_ document will _not_ trigger the ETL process.|
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/server/ongoing-tasks/etl/basics.markdown]: WRT "Note:  Changes made ..." - nice catch, very useful info for a reader

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/ravendb-etl-task.markdown [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+5. Select the collections for the ETL task -or - apply to all collections  
+{PANEL/}
+
+{PANEL: RavenDB ETL Task - Details in Tasks List View}
+
+![Figure 3. RavenDB ETL Task - Task List View](images/ravendb-etl-task-3.png "Tasks List View Details")
+
+1. **RavenDB ETL Task Details**:
+   *  Task Status - Active / Not Active / Not on Node / Reconnect ? (see issue RavenDB-10674)
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/ravendb-etl-task.markdown]: I wasn't aware we have such state as 'Reconnect'. I'll check what we can do here (I took RavenDB-10674)

**Comment by arekpalinski** [comment]
Awesome job! Thanks for great refactoring of existing etl articles

File: Documentation/4.0/Raven.Documentation.Pages/server/ongoing-tasks/etl/basics.markdown (hunk 1 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/server/ongoing-tasks/etl/basics.markdown]: done

**Pull request merged**